res/openvpn-gui-res-it.rc

```diff
@@ -327,165 +342,164 @@ Quest'azione può richiedere privilegi amministrativi."
-    IDS_ERR_STOP_SERV_OLD_VER "Non puoi usare OpenVPN GUI per far partire una connessione mentre il servizio di OpenVPN è in funzionamento (con OpenVPN 1.5/1.6). Arresta il servizio di OpenVPN se vuoi usare OpenVPN GUI."
+    IDS_ERR_ONE_CONN_OLD_VER "Quando usi una versione di OpenVPN più vecchia della 2.0-beta6 puoi avere una sola connessione attiva alla volta."
+    IDS_ERR_STOP_SERV_OLD_VER "Con OpenVPN 1.5/1.6 non puoi usare OpenVPN GUI per far partire una connessione mentre il servizio di OpenVPN è in funzione.\nSe vuoi usare OpenVPN GUI arresta il servizio di OpenVPN ."
     IDS_ERR_CREATE_EVENT "Il creatore di eventi è fallito all'uscita dell'evento: %ls"
-    IDS_ERR_UNKNOWN_PRIORITY "Nome di priorità sconosciuto: %ls"
-    IDS_ERR_LOG_APPEND_BOOL "Il file di log ha aggiunto un puntatore (dato come '%ls') deve essere '0' o '1'"
-    IDS_ERR_GET_MSIE_PROXY "Non riesco ad utilizzare la configurazione di MSIE per il proxy."
+    IDS_ERR_UNKNOWN_PRIORITY "Nome priorità sconosciuto: %ls"
+    IDS_ERR_LOG_APPEND_BOOL "Il file registro ha aggiunto un puntatore (dato come '%ls') deve essere '0' o '1'"
+    IDS_ERR_GET_MSIE_PROXY "Impossibile usare la configurazione di MSIE per il proxy."
     IDS_ERR_INIT_SEC_DESC "InitializeSecurityDescriptor fallito."
     IDS_ERR_SET_SEC_DESC_ACL "SetSecurityDescriptorDacl fallito."
     IDS_ERR_CREATE_PIPE_OUTPUT "CreatePipe on hOutputWrite fallito."
     IDS_ERR_CREATE_PIPE_INPUT "CreatePipe on hInputRead fallito."
     IDS_ERR_DUP_HANDLE_OUT_READ "DuplicateHandle on hOutputRead fallito."
     IDS_ERR_DUP_HANDLE_IN_WRITE "DuplicateHandle on hInputWrite fallito."
-    IDS_ERR_CREATE_PROCESS "CreateProcess fallito, exe='%ls' cmdline='%ls' dir='%ls'"
+    IDS_ERR_CREATE_PROCESS "CreateProcess fallito, exe='%ls' linea di comando='%ls' cartella='%ls'"
     IDS_ERR_CREATE_THREAD_STATUS "CreateThread fallito per visualizzare lo stato."
-    IDS_NFO_STATE_WAIT_TERM "Stato corrente: In attesa che OpenVPN termini…"
-    IDS_NFO_STATE_CONNECTED "Stato corrente: Connesso"
+    IDS_NFO_STATE_WAIT_TERM "Stato attuale: in attesa che OpenVPN termini…"
+    IDS_NFO_STATE_CONNECTED "Stato attuale: connesso"
     IDS_NFO_NOW_CONNECTED "%ls è ora connesso."
     IDS_NFO_ASSIGN_IP "IP assegnato: %ls"
-    IDS_ERR_CERT_EXPIRED "Non riesco a connettermi perché il tuo certificato è scaduto o l'orologio di sistema non è corretto."
-    IDS_ERR_CERT_NOT_YET_VALID "Non riesco a connettermi perché il tuo certificato non è ancora valido. Assicurati che l'orologio di sistema sia corretto."
-    IDS_NFO_STATE_RECONNECTING "Stato corrente: Riconnessione in corso"
-    IDS_NFO_STATE_DISCONNECTED "Stato corrente: Disconnesso"
+    IDS_ERR_CERT_EXPIRED "Impossibile connettersi perché il certificato è scaduto o l'ora di sistema non è corretta."
+    IDS_ERR_CERT_NOT_YET_VALID "Impossibile connettersi perché il certificato non è ancora valido. Assicurati che l'ora di sistema sia corretta."
```

Comment:
Was this resolved?

Reply:
I saw a reviewer comment on this one in the previous version, but no response from you. So the question was is this change as discussed?
I'm in no way qualified to review this, so I was only making it easy for you and @ordex by going through the previous review comments and how they were resolved. I noticed a number of them resolved in some way but the resolution not reflected in this updated version.
Anyway, you can push a fixup commit here for any further changes you want to make instead of a new a PR. I can squash them at merge time.

Reply:
For me now is ok.